version.properties

```diff
@@ -1,4 +1,4 @@
 #Thu Jan 23 09:06:23 UTC 2025
-dependency.coreSdk.version=2.0.0
+dependency.coreSdk.version=2.0.1
 widgets.versionCode=88
```

Comment:
`versionCode` shall also be incremented, isn't it?

Reply:
We will do it during the upgrade to the 3.0.0 version in widgets.